test/common_test/CMakeLists.txt

```diff
@@ -7,12 +7,11 @@ set(tests
 link_directories(${PROJECT_BINARY_DIR}/test)
 
 function(configure_common_test PHYSICS_ENGINE_NAME test_name)
-  set(_env_vars)
-  add_test(NAME ${test_name}_${PHYSICS_ENGINE_NAME} COMMAND ${test_name})
-  list(APPEND _env_vars "LIB_TO_TEST=${CMAKE_BINARY_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}${PROJECT_LIBRARY_TARGET_NAME}-${PHYSICS_ENGINE_NAME}-plugin${CMAKE_SHARED_LIBRARY_SUFFIX}")
-  set_tests_properties(${test_name}_${PHYSICS_ENGINE_NAME}
-    PROPERTIES
-      ENVIRONMENT "${_env_vars}")
+  add_test(NAME ${test_name}_${PHYSICS_ENGINE_NAME}
+    COMMAND
+      ${test_name}
+      ${CMAKE_BINARY_DIR}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}${PROJECT_LIBRARY_TARGET_NAME}-${PHYSICS_ENGINE_NAME}-plugin${CMAKE_SHARED_LIBRARY_SUFFIX}
```

Comment:
nit: use a generator expression to get the library file path from a target name

Reply:
https://github.com/gazebosim/gz-physics/pull/359/commits/2d5ba810cab566e00a96f8292799f1c4dad0c365